picard/ui/itemviews.py

```diff
@@ -399,8 +400,7 @@ def _add_other_versions():
                     else:
                         action_menu = plugin_menus[key] = action_menu.addMenu(key[-1])
                 action_menu.addAction(action)
-
-        if isinstance(obj, Cluster) or isinstance(obj, ClusterList) or isinstance(obj, Album):
+        if (isinstance(obj, Cluster) and len(obj.files) > 0) or (isinstance(obj, ClusterList) and len(obj) > 0) or isinstance(obj, Album):
```

Comment:
You might wanna break up the if logic into multiple lines. This is hard to read.